### PR TITLE
CVE-2009-2902

### PR DESCRIPTION
**Pull request by cgs8601**
CVE-2009-2902 - Curtis

### CODE REVIEW

File: cves/CVE-2009-2902.yml [hunk 1]
```diff
@@ -204,8 +204,9 @@ interesting_commits:
     emerging themes?
     If there are no interesting commits, demonstrate that you completed this
     section by explaining what happened between the VCCs and the fix.
-  answer: There were no changes related to the vulnerability until the fixing commit. It appears that the fixing commit
-    fixed more than one CVE, and this vulnerability was not an extremely high priority one.
+  answer: 
+    The issue was only addressed in the 6.0.X and 5.5.X commits. 
+    I did not find any other commits relating to this issue.
```
Comment (gollb310) [inline cves/CVE-2009-2902.yml]: Were there any interesting changes to the file in general unrelated to the vulnerability? If not, just give a summary of some of the commits to show you went through them.

File: cves/CVE-2009-2902.yml [hunk 2]
```diff
@@ -43,15 +43,14 @@ description_instructions: |
...
-  potentially overwrite files on the system running Apache Tomcat under the
-  "work/<engine-name>/<hostname>" directory. This vulnerability is an example of
-  improper limitation of a pathname to a restricted directory because a user without
-  access to the system can violate the integrity of files they should not have access
-  to.
-
+description:
+  A path traversal bug allowed a user to specify a directory
+  to search for the a file in, and would delete the specified directory
+  as part of failure cleanup if the file had an invalid filename.
```
Comment (gollb310) [inline cves/CVE-2009-2902.yml]: The grammar for your first description paragraph feels a bit clunky and needs some rewording. things like "to search for the a file in" doesn't make much sense.

**Review by gollb310 — COMMENTED** [review]
Overall the investigation and report were well done and thoroughly documented, with a bit of grammar rework and some small expansion needed. +3 Upvotes.

File: cves/CVE-2009-2902.yml (hunk 2 above)
Comment (cgs8601) [reply cves/CVE-2009-2902.yml]: Fixed.

File: cves/CVE-2009-2902.yml (hunk 1 above)
Comment (cgs8601) [reply cves/CVE-2009-2902.yml]: Fixed.

**Pull request merged**